tests/core/test_key_api_proxy_methods.py

```diff
@@ -2,11 +2,11 @@
 
 from eth_utils import (
     keccak,
-    ValidationError,
 )
 
 from eth_keys import KeyAPI
 from eth_keys.backends import NativeECCBackend
+from eth_keys.exceptions import ValidationError
```

Comment:
Maybe better to have these import directly from `eth_utils`?

Reply:
:+1: I was using it to verify that the eth_keys exception import still works. But, better to do that explicitly with a new test, instead.